org.wso2.carbon.metadata.server/src/main/java/org/wso2/carbon/metadata/server/dao/jdbc/JDBCMetadataDAO.java

```diff
@@ -16,10 +16,47 @@
 
 package org.wso2.carbon.metadata.server.dao.jdbc;
 
+import org.wso2.carbon.metadata.server.api.Key;
+import org.wso2.carbon.metadata.server.api.MetadataStoreException;
+import org.wso2.carbon.metadata.server.api.Resource;
 import org.wso2.carbon.metadata.server.dao.MetadataDAO;
+import org.wso2.carbon.metadata.server.impl.CollectionImpl;
+
+import java.util.ArrayList;
+import javax.xml.ws.http.HTTPException;
 
 /**
  * JDBC Metadata DAO
  */
 public class JDBCMetadataDAO implements MetadataDAO {
+    @Override
+    public void add(Resource resource) throws MetadataStoreException {
```

Comment:
What would be the purpose of the class ? would it make sense to throw NotImplementedException ?